test/hotspot/jtreg/vmTestbase/nsk/jdi/VirtualMachine/dispose/dispose003a.java

```diff
@@ -128,16 +128,18 @@ public static void main (String argv[]) {
                          while (true) {
                              instruction = pipe.readln();
                              if (instruction.equals("check_done")) {
-                                 if (test_thread.isAlive()) {
+                                 if (Utils.isAlive(test_thread)) {
                                      logErr("ERROR: thread2 thread is still alive");
                                      exitCode = FAILED;
                                  }
                                  break;
                              } else if (instruction.equals("check_alive")) {
                                  log1("checking on: thread2.isAlive");
-                                 if (test_thread.isAlive()) {
-                                     test_thread.resume();
+                                 if (Utils.isAlive(test_thread)) {
                                      pipe.println("alive");
+                                     logErr("ERROR thread is alive after vm.dispose()");
```

Comment:
You've added a nice diagnostic line here for this case of the thread
being unexpectedly alive, but you didn't do so for the other test cases
that you've updated in v01. Is this log mesg not applicable in those cases?

Reply:
Added to all tests.